api/proto/evecommon/devmodelcommon.proto

```diff
@@ -19,6 +19,9 @@ enum PhyIoType {
   PhyIoNetWLAN = 5;
   PhyIoNetWWAN = 6;
   PhyIoHDMI = 7;
+  // enum 8 is reserved for backward compatibility with controller API
+  PHY_IO_TYPE_STORAGE_NVME_DEVICE = 9;
+  PHY_IO_TYPE_STORAGE_SATA_DEVICE = 10;
```

Comment:
Sorry missed there was an update to look at.
Even though yetus complains, when adding enum values to an enum type we have kept the same style as the existing values. (Alternatively would be to separately change allow of them).
Thus callling these PhyIoSataDevice etc makes the most sense.

Reply:
OK I revert back. I was trying to satisfy Yetus and assumed its ignoring earlier enums because it was told you for legacy reasons.

Reply:
Updated the commits.

Reply:
Note that you also have conflicting files to regenerate before I can merge this.

Reply:
If you are asking about the generated files, yes I did run "make proto" and "make proto-vendor" and updated those files in the commit.